libvcx/Cargo.toml

```diff
@@ -45,10 +45,10 @@ regex = "1.1.0"
 rust-base58 = "0.0.4"
 rmp-serde = "0.13.7"
 base64 = "0.8.0"
-openssl = { version = "0.10.29" }
+openssl = { version = "0.10.35", features = ["vendored"] }
```

Comment:
I was looking at this and got concerned about this

> The vendored copy will not be configured to automatically find the system’s root certificates, but the openssl-probe crate can be used to do that instead.


So I went through actual usage of openssl crate and it turn out it *might* be an overkill. What's being used is:
```
use openssl::sha::sha256
use openssl::bn::BigNum;
```
I somewhat presumed `openssl` is there because `reqwest` needs it, but `reqwest` comes has its own `tls` dependency https://github.com/seanmonstar/reqwest/blob/master/Cargo.toml

So out openssl usage is not actually ever dealing with certificates, so the quote on top of this comment doesn't matter. I think ultimate solution will be to get rid of openssl dependency. 
Perhaps:
-  https://github.com/RustCrypto/hashes for Sha256 
- https://github.com/rust-num/num-bigint for big integer